lldb/source/Plugins/SymbolFile/DWARF/SymbolFileDWARFDebugMap.cpp

```diff
@@ -716,6 +718,27 @@ bool SymbolFileDWARFDebugMap::ParseDebugMacros(CompileUnit &comp_unit) {
   return false;
 }
 
+void SymbolFileDWARFDebugMap::ForEachSymbolFile(
+    std::string description,
+    std::function<IterationAction(SymbolFileDWARF &)> closure) {
+  const size_t num_oso_idxs = m_compile_unit_infos.size();
+  Progress progress(std::move(description), "", num_oso_idxs,
+                    /*debugger=*/nullptr,
+                    /*minimum_report_time=*/std::chrono::milliseconds(20));
+  for (uint32_t oso_idx = 0; oso_idx < num_oso_idxs; ++oso_idx) {
+    if (SymbolFileDWARF *oso_dwarf = GetSymbolFileByOSOIndex(oso_idx)) {
+      progress.Increment(oso_idx, oso_dwarf->GetObjectFile()
+                                      ? oso_dwarf->GetObjectFile()
+                                            ->GetFileSpec()
+                                            .GetFilename()
+                                            .GetString()
+                                      : "");
```

Comment:
Does this just show a path to the .a file for all .o files within a .a file? We might want to check the object name in the lldb_private::Module:
```
ConstString object_name = oso_dwarf->GetModule().GetObjectName();
```

Reply:
You're right. I created https://github.com/llvm/llvm-project/pull/133370